commons/src/main/java/org/infinispan/commons/event/KVPEvent.java

```diff
@@ -0,0 +1,30 @@
+package org.infinispan.commons.event;
+
+import java.io.Serializable;
+
+public class KVPEvent<K, V> implements Serializable {
```

Comment:
I would have to agree on all points, and I would say it wouldn't hurt to add the equals and hashCode as well (IDE does it quick enough anyways).

Also I would say to not make this implement Serializable.  If we have a use case that requires these to be serialized we should provide a proper Externalizer like we do for other classes.